lib/template/base.rb

```diff
@@ -48,10 +48,57 @@ def initialize(post_message_definitions:)
     @post_message_definitions = post_message_definitions
   end
 
+  # rubocop:disable Layout/CommentIndentation
   # rubocop:disable Lint/UselessMethodDefinition
   # @return [Binding]
   def binding
     super
   end
   # rubocop:enable Lint/UselessMethodDefinition
+  # rubocop:enable Layout/CommentIndentation
+
+private
+
+  # @return [Hash]
+  def entity_post_message_definitions
```

Comment:
All of these methods are needed by a new generator I'm working on, so moving them to the parent class. Maybe they'll get moved to a separate helper class but starting with moving them to the parent seems like a good start to me.